source/jormungandr/jormungandr/default_settings.py

```diff
@@ -54,6 +54,9 @@
     }
 }
 
+# Bike self-service configuration
+BSS_PROVIDER = []
```

Comment:
it's not a constant, in the real configuration file there will be several bss providers